imgui.lua

```diff
@@ -35,7 +35,7 @@ local function shouldAcceptInput()
 	end
 
 	-- don't process input if we're doing VGUI stuff (and not in context menu)
-	if vgui.CursorVisible() and vgui.GetHoveredPanel() ~= g_ContextMenu then
+	if vgui.CursorVisible() and vgui.GetHoveredPanel() ~= g_ContextMenu and not vgui.IsHoveringWorld() then
```

Comment:
This change seemed necessary because when calling `gui.EnableScreenClicker(true)` in my weapon to bring up the mouse, input did not seem to be accepted because I was hovering the world panel.